values.yaml

```diff
@@ -190,6 +190,27 @@ ui:
     # ServiceSpec object.
     additionalSpec: null
 
+  ## True if you want to create an Ingress for the Consul UI.
+  ingress:
+    enabled: false
+
+    # hosts is a list of host name to create Ingress rules.
+    # The value below is an array of objects, examples are shown below.
+    hosts: []
+     # - sslexample.foo.com
+
+    # tls is a list of hosts and secret name in an Ingress
+    # which tells the Ingress controller to secure the channel.
+    # The value below is an array of objects, examples are shown below.
+    tls: []
+     # - hosts:
+     #   - sslexample.foo.com
+     #   secretName: testsecret-tls
+
+    # This should be a multi-line string mapping directly to the a map of
+    # the annotations to configure some options depending on the Ingress controller
+    annotations: null
```

Comment:
IMO in most cases annotations default values is set to `{}` rather than `null`

Reply:
This chart uses multiline strings for values like this. Example: https://github.com/hashicorp/consul-helm/blob/master/values.yaml#L293